src/components/graph/menus/create-node-menu.js

```diff
@@ -90,6 +103,29 @@ const CreateNodeMenu = ({
             action: () => createNetworkModificationNode('AFTER'),
             id: 'insertNetworkModificationNodeBelow',
         },
+        COPY_MODIFICATION_NODE: {
+            onRoot: false,
+            action: () => copyNetworkModificationNode(),
+            id: 'copyNetworkModificationNode',
+        },
+        PASTE_MODIFICATION_NODE_ON_NEW_BRANCH: {
+            onRoot: false,
+            action: () => pasteNetworkModificationNode('NEW_BRANCH'),
+            id: 'pasteNetworkModificationNodeOnNewBranch',
+            disabled: selectedNodeForCopy == null,
```

Comment:
=== ?

Reply:
nah i want to test for undefined and null

Reply:
Ah yes